src/components/CardMenus/CardMenus.js

```diff
@@ -19,11 +18,20 @@ const CardMenu = () => {
 
   return (
     <div>
-      <MainTab />
-      <Grid className={classes.control} container elevation justifyContent="space-around" spacing={3}>
+      <Grid className={classes.control} container elevation={3} spacing={3}>
         {UserData.map((usedata) => {
           const { id, name, price, resturantName, img } = usedata;
-          return <CommonCard key={id} img={img} name={name} price={price} resturantName={resturantName} />;
+
+          return (
+            <CommonCard
```

Comment:
Shouldn't the card name be Recipe card or food card because that's not a common card it is specific to recipe or food.